src/libraries/Native/AnyOS/System.Globalization.Native/pal_locale.h

```diff
@@ -0,0 +1,15 @@
+// Licensed to the .NET Foundation under one or more agreements.
```

Comment:
I don't know why this file wasn't marked as renamed as I used git mv to move all the files. But this file contains a few edits from the original one. Only changed "pal_compiler.h" to "compiler.h" and added `EXTERN_C` to the declarations.